lightning/src/ln/channelmanager.rs

```diff
@@ -2940,77 +2934,6 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	// smaller than 100:
 	const STATIC_ASSERT: u32 = Self::HALF_MESSAGE_IS_ADDRS - 100;
 
-	/// Regenerates channel_announcements and generates a signed node_announcement from the given
-	/// arguments, providing them in corresponding events via
-	/// [`get_and_clear_pending_msg_events`], if at least one public channel has been confirmed
-	/// on-chain. This effectively re-broadcasts all channel announcements and sends our node
-	/// announcement to ensure that the lightning P2P network is aware of the channels we have and
-	/// our network addresses.
-	///
-	/// `rgb` is a node "color" and `alias` is a printable human-readable string to describe this
-	/// node to humans. They carry no in-protocol meaning.
-	///
-	/// `addresses` represent the set (possibly empty) of socket addresses on which this node
-	/// accepts incoming connections. These will be included in the node_announcement, publicly
-	/// tying these addresses together and to this node. If you wish to preserve user privacy,
-	/// addresses should likely contain only Tor Onion addresses.
-	///
-	/// Panics if `addresses` is absurdly large (more than 100).
-	///
-	/// [`get_and_clear_pending_msg_events`]: MessageSendEventsProvider::get_and_clear_pending_msg_events
-	pub fn broadcast_node_announcement(&self, rgb: [u8; 3], alias: [u8; 32], mut addresses: Vec<NetAddress>) {
```

Comment:
`channelmanager.rs` L:2928 there's a `broadcast_node_ann` reference that may want to be moved to `peer_handler`